ci/define_pvs.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: "Define Pvs"
+  hosts: controller
+  vars:
+   base_dir: "{{ ansible_user_dir }}"
+  tasks:
+    - name: "Log into the cluster"
+      ansible.builtin.import_role:
+        name: rhol_crc
+        tasks_from: add_crc_creds.yml
+
+    - name: Check if the install_yamls already exists
+      ansible.builtin.stat:
+        path: "{{ base_dir }}/install_yamls"
+      register: check_install_yamls_dir
+
+    - name: "Get Install yaml repo"
+      when: not check_install_yamls_dir.stat.exists
+      ansible.builtin.git:
+        repo: https://github.com/openstack-k8s-operators/install_yamls
+        dest: "{{ base_dir }}/install_yamls"
+   
+    - name: Check if make command exists
+      ansible.builtin.command: "command -v make"
+      register: make_check_result
+      ignore_errors: true
+
+    - name: Install make if not present
+      ansible.builtin.package:
+        name: make
+        state: present
+      when: make_check_result.rc != 0
+
+    - name: Creating the Pvs 
+      ansible.builtin.shell: make crc_storage
+      args:
+        chdir: "{{ base_dir }}/install_yamls/"
+
+    - name: Print make result
+      ansible.builtin.debug:
+        var: make_output
+
+    - name: Confirm  Pvs exist
+      ansible.builtin.shell: 
+        cmd: |
+          oc get pv | grep "storage..-crc"| wc -l
+      register: cmd_output
+      failed_when: "cmd_output.stdout|int !=12"
```

Comment:
I'd like to see some output here as well e.g. a debug.
You can use the `ansible.builtin.fail` module to check `cmd_output.stdout_lines | length`

Since we don't control the creation of the PVs, I would revise this test, since the number of lines may change without changing the PVs, for example.

Reply:
IINM, we want to be able to use the PVs in the ``{{ namespace }}`` namespace/project [1], so there should be a check for that.

[1] https://github.com/infrawatch/service-telemetry-operator/blob/77dea87dec0656c48d0c7e0b9a09d0fe826425aa/ci/vars-zuul-common.yml#L2

Reply:
yes , this is not the final test , I believe I still need to create pvc to be used directly in the pods this create only the pvs , but wanted to check how the zuul will run it , so I created it as a draft pull request but this is not done yet